script/dryrun

```diff
@@ -26,6 +26,9 @@ DERIVED_DATASETS_DRY_RUN_URL = (
     "https://us-central1-moz-fx-data-derived-datasets.cloudfunctions.net/"
     "bigquery-etl-dryrun"
 )
+PUBLIC_DATASETS_DRY_RUN_URL = (
+    "https://us-central1-mozilla-public-data-prod.cloudfunctions.net/bigquery-etl-dryrun"
```

Comment:
This is just a placeholder for now and it needs to be verified if this is actually the URL once the project is created

Reply:
Do we need a separate dry run endpoint for public tables? I would imagine we could still use the shared-prod version to access public data.

Reply:
Looking a bit more into this, it should be possible, however it would require to adjust the `bigquery-etl-dryrun` Cloud Function to allow specifying different projects. The public data queries need to get executed in the public data project.